seq2science/rules/trackhub.smk

```diff
@@ -556,6 +564,27 @@ if config.get("create_trackhub"):
                             out["files"].append(file)
                             signal_view.add_tracks(track)
 
+                            # add the control sample(s)
+                            if "control" in treps and isinstance(treps.loc[trep, "control"], str):
+                                control_name = treps.loc[trep, "control"]
+
+                                file = f"{config['bigwig_dir']}/{assembly}-{control_name}.{peak_caller_sorter}-{peak_caller_sort_order}.bw"
+                                priority += 1.0
+                                track = trackhub.Track(
+                                    name=trackhub.helpers.sanitize(f"{rep_to_descriptive(trep)}_control bw"),
+                                    tracktype="bigWig",
+                                    short_label=f"{shorten(rep_to_descriptive(trep), 14-len(pcs), ['signs', 'vowels', 'center'])}_control{pcs} bw",  # <= 17 characters suggested
+                                    long_label=f"{rep_to_descriptive(trep)}_control{peak_caller_suffix} bigWig",
+                                    subgroups={"conditions": safedescr},
```

Comment:
we could add an additional "controls" subgroup

Reply:
Something like:

`subgroups={"conditions": safedescr, "input_control": True},` (or input_control False for the others?)

Reply:
I'm pretty sure you have more knowledge about the trackhub and how it works than me btw :smile: